data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+			`	Eventually Mr. Yang returns. "So sorry for that. I hope you were looked after suitably while you waited."`
+			`	Once you nod your assent, he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official "line" is that the government does not trade in rumors. However, an appropriate response is clearly needed. Republic Intelligence briefed the Cabinet of the Free Worlds Senate just yesterday. They have demanded that they have a diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you to meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`
+			choice
+				`	"Yes."`
+					goto conclude
+				`	"Alastair?"`
+			`	"Alastair Korban is the Syndicate CEO, naturally the CEO has some involvement here, but I trust that won't be a problem?"`
+			choice
+				`	"It should be fine."`
+			label conclude
+			`	"Excellent. The situation is progressing quickly and plans are still being made, so I am not entirely sure if it is they themselves who will be coming with us. Alastair is likely too busy to come personally, but we'll find out when we get there. In any case I think we should go to Deep first for speed, but you are the captain. Shall we be off?"`
+				accept
+	on visit
+		dialog phrase "generic stopover on visit"
+	on complete
+		log `Brought human diplomatic representatives from each human faction to Hai-home to address the crisis. Something drastic needs to be done to keep the secret from entirely bursting, but they know how long they will have to work with. Asked to rout pirates hoping to benefit from the chaos while they work on a solution.`
+		conversation
+			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
+			`	You get down to business when Sayari inquires how your journey went.`
+			choice
+				`	"I don't recall there being nearly so many pirates."`
+			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
+			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
+			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`
```

Comment:
This more a matter of it being not very kosher (from an international relations pov) to have independent agreements with dissident elements of a polity that you otherwise maintain an official relationship with.
This is an expression of surprise that the Hai treat with pirates as separate entities to the Republic, rather than treating with the Republic to handle them. The arrangement runs counter to conventional international relations policy and legitimises the independence of the pirates - something not usually accepted by government's like the Republic.

Alondo is from a place very far-removed from the Hai, this arrangement would be surprising. Especially as (by convention) it implies that between the Republic and the Hai, the Hai are the stronger of the two.